lib/rucio/tests/test_bin_rucio.py

```diff
@@ -2327,3 +2331,22 @@ def test_admin_rse_update_unsupported_option(self):
         print(out, err)
         assert exitcode == 0
         assert not err
+
+    @pytest.mark.noparallel(reason='Modify config')
+    def test_lifetime_cli(self):
+        """ CLIENT(USER): Check CLI to declare lifetime exceptions """
+        # Setup data for CLI check
+        tmp_dsn_name = 'container' + rse_name_generator()
+        tmp_dsn_did = self.user + ':' + tmp_dsn_name
+        self.did_client.add_did(scope=self.user, name=tmp_dsn_name, did_type='DATASET')
+        self.did_client.set_metadata(scope=self.user, name=tmp_dsn_name, key='eol_at', value=(datetime.utcnow() - timedelta(seconds=86400)).strftime('%Y-%m-%d'))
+        tomorrow = datetime.utcnow() + timedelta(days=1)
+        tomorrow = tomorrow.strftime('%Y-%m-%d')
+        self.config_client.set_config_option(section='lifetime_model', option='cutoff_date', value=tomorrow)
+        string = codecs.encode("%s\n" % tmp_dsn_did, 'utf-8')
```

Comment:
Could you explain the necessity for using the `codecs` module?

Reply:
Removed